rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.76.0"
```

Comment:
Ah, because the canister are getting built. But this would have been needed before?
What about specifying the newer `1.77.2` version?

Reply:
Yes, this would have been needed before but because my environment already had the tools I didn't realize it.

I realized it when it had to be built on CI.

Reply:
Done! Set to 1.77.2